flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/join/ProcTimeIntervalStreamJoin.java

```diff
@@ -24,13 +24,13 @@
 import org.apache.flink.table.runtime.typeutils.RowDataTypeInfo;
 
 /**
- * The function to execute processing time bounded stream inner-join.
+ * The function to execute processing time interval stream inner-join.
  */
-public final class ProcTimeBoundedStreamJoin extends TimeBoundedStreamJoin {
+public final class ProcTimeIntervalStreamJoin extends TimeIntervalStreamJoin {
 
```

Comment:
What is a interval stream join ?

Reply:
In order to distinguish with batch join implementation, because they are located in the same package.

Reply:
Then `ProcTimeStreamIntervalJoin` sounds better.

Reply:
Let's call it `ProcTimeIntervalJoin`, I will move them into a specific `interval` package. All the implementations under this package should belong to streaming.